frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -212,6 +214,15 @@ export const createOrUpdateDeployment = (
                   },
                 }),
               },
+              ...(readinessProbe.enabled
+                ? { readinessProbe: constructProbeData(readinessProbe.data) }
+                : {}),
+              ...(livenessProbe.enabled
+                ? { livenessProbe: constructProbeData(livenessProbe.data) }
+                : {}),
+              ...(startupProbe.enabled
+                ? { startupProbe: constructProbeData(startupProbe.data) }
+                : {}),
```

Comment:
I have seen this same code in 4 places. Create a utility for this that will return the required data. Something like this
`getProbeData(readinessProbe, livenessProbe, startupProbe)`.